packages/adapter-netlify/index.js

```diff
@@ -54,15 +54,16 @@ export default function ({ split = false, edge = edge_set_in_env_var } = {}) {
 			builder.log.minor(`Publishing to "${publish}"`);
 
 			builder.log.minor('Copying assets...');
-			builder.writeClient(publish);
-			builder.writePrerendered(publish);
+			const prefixedPublish = `${publish}${builder.config.kit.paths.base}`;
+			builder.writeClient(prefixedPublish);
+			builder.writePrerendered(prefixedPublish);
```

Comment:
this should use underscores since it's internal
```suggestion
			const publish_dir = `${publish}${builder.config.kit.paths.base}`;
			builder.writeClient(publish_dir);
			builder.writePrerendered(publish_dir);
```